css-two-column-layout/styles.css

```diff
@@ -0,0 +1,80 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0 auto;
+  font-family: Karla, sans-serif;
+  line-height: 150%;
+}
+
+h1,
+h3 {
+  text-transform: uppercase;
+}
+
+.header {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 32px;
+}
+
+.full-colum-head {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding: 16px;
+}
+
+.head-text {
+  text-align: center;
+  padding: 16px 100px 0;
+}
+
+.content {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 0 50px;
+  font-weight: lighter;
+}
+
+.row {
+  display: flex;
+}
+
+.two-column {
```

Comment:
make sure the column has a set width on it, don't let flexbox determine the size!